msbuild/Makefile

```diff
@@ -193,7 +199,7 @@ endif
 ##
 
 ##
-## XI ##
```

Comment:
There are a number of unnecessary diffs like this. Please revert them. Reach out to me on discord and I'll show you how to do so easily.